src/dos/program_more.cpp

```diff
@@ -226,11 +226,11 @@ void MORE::AddMessages()
 	MSG_Add("PROGRAM_MORE_NEW_DEVICE",
 	        "[reset][color=light-yellow]--- device %s ---[reset]");
 	MSG_Add("PROGRAM_MORE_PROMPT_SINGLE",
-	        "[reset][color=light-yellow]--- press SPACE or ENTER for more ---[reset]");
```

Comment:
I think that's unnecessary because nobody uses the 40-column modes willingly 😄 Then all the help texts and config setting descriptions assume 80 columns at least, the output of the mixer command too, and probably a few other things. I except most things to break in 40-column modes, in other words, and I'm fine with that 😄

Reply:
> OK, so feel free to merge it as is.
> 
> (offtopic) The second screenshot looks very intriguing :)

Yeah, I've started the "preparations" 😄